### DISCUSSION TITLE
[RFC] 071 - 多模态生成 (绘图/视频/音乐)

### DISCUSSION THREAD

**Discussion by canisminor1990** [lobehub/lobehub#4741] [RFCs]
## 预备特性 

- [ ] 提供更符合特定创作场景的模态面板
	- 更丰富的侧边栏配置
	- 提供 Artifacts 和 专业 模式
- [x] 同时内置多模态 Provider 配置
	- 按模态类型进行聚合
- [ ] 提供系统助手中多模态的模型配置
- [x] 补充 `@lobehub/icons` https://github.com/lobehub/lobe-icons/commit/dce7dd1811158c6dad6a670cf6ebf599bdb08eec

## 预备接入模型服务

**绘图**：  https://github.com/lobehub/lobe-chat/discussions/7442
- [x] ~~MidJourney~~
- [x] Stable Diffusion
- [x] Flux
- [x] Dall-E-3
- [x] CogView (智谱)

**视频**
- [ ] Luma
- [ ] Runway
- [ ] Kling (可灵)
- [ ] Viggle
- [ ] CogVideo (智谱)

**音乐**
- [ ] Suno
- [ ] ?CogSound (智谱)

**Comment by arvinxx** [comment]
参考：

<img width="1719" alt="image" src="https://github.com/user-attachments/assets/f24a3a55-9b28-4d79-a1e5-29ec7abced4e">


![image](https://github.com/user-attachments/assets/b704cf06-fac6-4ccd-a830-1ab1ac5bcdbc)

![image](https://github.com/user-attachments/assets/6c0c0f18-7df7-4881-9905-773ab06ab87d)

![image](https://github.com/user-attachments/assets/3f136f21-a3df-4d3f-94b2-49846e5da305)

**Comment by BrandonStudio** [comment]
Cloudflare Text-to-Image models (https://github.com/lobehub/lobe-chat/discussions/3186).

**Comment by m0eak** [comment]
这个大概什么时候上啊

**Comment by ccbbq9527** [comment]
求问这个有计划开发吗

**Comment by Wool-yang** [comment]
有考虑把 openai 的 image api 整合一下吗

**Reply by ONLY-yours** [reply]
有了，现在绘画的多模态单独开了一个 Tab 
![Uploading image.png…]()
